scripts/fix-code-style.sh

```diff
@@ -7,5 +7,5 @@ echo #SCRIPTPATH
 cd $SCRIPTPATH/..
 
 find . -iname "*.h" -o -iname "*.cpp" | \
-egrep -v "^(./comms/mavlink/v1.0/|./boards/airbourne/airbourne/|./boards/breezy/breezystm32|./.git|./test/build)" | \
+egrep -v "^(./comms/mavlink/v1.0/|./boards/airbourne/airbourne/lib/|./boards/breezy/breezystm32|./.git|./test/build)" | \
```

Comment:
Shouldn't we ignore the whole airbourne directory, since it's a submodule and it's formatting is handled independently?

Reply:
There isn't any formatting information in airbourne. Additionally, IDE's don't necessarily differentiate between firmware and airbourne.

That would be cleaner. This was mostly a quick fix so that PR's wouldn't have a ton of whitespace changes.